cvat/apps/dataset_manager/views.py

```diff
@@ -127,14 +176,55 @@ def export_project_annotations(project_id, dst_format=None, server_url=None):
     return export(dst_format, project_id=project_id, server_url=server_url, save_images=False)
 
 
-def clear_export_cache(file_path, file_ctime, logger):
+def clear_export_cache(file_path: str, file_ctime: float, logger: logging.Logger) -> None:
     try:
-        if osp.exists(file_path) and osp.getctime(file_path) == file_ctime:
+        with get_dataset_cache_lock(
+            file_path,
+            block=True,
+            acquire_timeout=EXPORT_CACHE_LOCK_TIMEOUT,
+            ttl=rq.get_current_job().timeout,
+        ):
+            if 'job' in file_path:
+                instance_type = 'job'
+            elif 'task' in file_path:
+                instance_type = 'task'
+            elif 'project' in file_path:
+                instance_type = 'project'
+            else:
+                assert False
+
+            cache_ttl = get_export_cache_ttl(instance_type)
+
+            instance_timestamp = None
+            with suppress(AttributeError): # backward compatibility
+                instance_timestamp = get_file_instance_timestamp(file_path)
+            if instance_timestamp and instance_timestamp != file_ctime:
```

Comment:
Seems like the second half of this condition would always be false, because the `file_path` passed to the function is derived from the `file_ctime` passed to the function.

Reply:
Updated, removed the check.